py/ai/h2o/sparkling/ml/features/H2OTargetEncoder.py

```diff
@@ -92,15 +92,15 @@ def setBlendedAvgEnabled(self, value):
 
     def setBlendedAvgInflectionPoint(self, value):
         assert_is_type(value, int, float)
-        return self._set(blendedAvgInflectionPoint=value)
+        return self._set(blendedAvgInflectionPoint=float(value))
```

Comment:
We need to explicitly convert the value to float even in setters, for the same reason as we do in constructor